<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperUserBoard</class>
 <widget class="QFrame" name="SuperUserBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QTabWidget" name="tab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>480</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="buy_book">
    <attribute name="title">
     <string>购书</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>书    名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>作    者</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>145</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数    量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>180</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>出版社</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_author">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_publishing_company">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>220</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>简    介</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="t_introduction">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>350</width>
       <height>79</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_confirm">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="number">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>I S B N</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_isbn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QComboBox" name="category_box">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>145</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>种    类</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="user">
    <attribute name="title">
     <string>用户</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>37</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string> 请输入你要查询的用户ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_user">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>33</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="user_info">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>90</y>
       <width>450</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="user_borrow_info">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>390</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>查看借阅信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modify_user_info">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>390</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>修改读者信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="book">
    <attribute name="title">
     <string>图书</string>
    </attribute>
    <widget class="QLineEdit" name="t_isbn_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>37</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>图书ISBN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN:</string>
     </property>
    </widget>
    <widget class="QTableView" name="book_info">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>90</y>
       <width>450</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_book">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>33</y>
       <width>80</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_borrow_info">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>390</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>查看借阅信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modify_book_info">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>390</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>修改书籍信息</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
